### ISSUE TITLE
Switch keyplane failed when the responder is a UITextField with secureTextEntry enabled and the language setting is non-English.

### ISSUE THREAD

**Issue by NSFish** [google/EarlGrey#47]
The switch keyplane key's accessibilityLabel on Software keyboard triggered by normal UITextField is "more, numbers" or "more, letters" in any language setting.
But with secureTextEntry enabled,  the accessibilityLabel of the key will be localized, which failed every test related to this. 
Currently I'm using KIF's mechanism  to bypass this, which directly calls the first responder view's "setText:" method.

Great solution for UITesting，thx:)

**Comment by khandpur** [comment]
Thanks for reporting this @NSFish 

The bug you reported seems very similar to an existing one: https://github.com/google/EarlGrey/issues/31 . Would you say there's any other additonal information that should be added there?

**Comment by NSFish** [comment]
Nope. @khandpur

**Comment by khandpur** [comment]
Marking this as a duplicate of #31
